index.bs

```diff
@@ -1406,7 +1406,7 @@ Note: The {{XRView/transform}} can be used to position camera objects in many re
 
 The <dfn method for="XRView">requestViewportScale(|scale|)</dfn> method requests that the user agent should set the [=view/requested viewport scale=] for this viewport to the requested value.
 
-When this method is invoked on a [=view=] |view|, the user agent MUST run the following steps:
+When this method is invoked on an {{XRView}} |view|, the user agent MUST run the following steps:
```

Comment:
you still need to talk about `|view|'s [=XRView/view=]` below, since the min/requested viewport scale are properties of the `[=view=]`

Reply:
Updated, I had missed that distinction. I'm now using separate `|xrview|` and `|view|` variables to differentiate them.